### ISSUE TITLE
"File not found" error with Flutter 3.10

### ISSUE THREAD

**Issue by sderungs99** [simolus3/sqlite3.dart#162]
Hi,

I am getting following error when adding this package to pubspec.yaml, after upgrading to Flutter 3.10. I can also replicate the same on the default Counter app when creating a new project and adding this library only. This is the full error I am getting when trying to build the project:

```zsh
Launching lib/main.dart on iPhone 14 Pro Max in debug mode...
main.dart:1
Upgrading Pods-Runner-frameworks.sh
Xcode build done.                                            3.1s
Failed to build iOS app
Error (Xcode): File not found: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/arc/libarclite_iphonesimulator.a

Error (Xcode): Linker command failed with exit code 1 (use -v to see invocation)

Could not build the application for the simulator.
Error launching application on iPhone 14 Pro Max.
Exited
```

Here is my pubspec.yaml:
```yml
name: dart3_app
description: A new Flutter project.
publish_to: 'none' # Remove this line if you wish to publish to pub.dev
version: 1.0.0+1

environment:
  sdk: '>=3.0.0 <4.0.0'
dependencies:
  sqlite3_flutter_libs: ^0.5.0
  flutter:
    sdk: flutter
  cupertino_icons: ^1.0.2

dev_dependencies:
  flutter_test:
    sdk: flutter
  flutter_lints: ^2.0.0
flutter:
  uses-material-design: true
```

Would be great if someone can look into this or let me know what I am doing wrong :-)

Cheers,
Sascha

**Comment by simolus3** [comment]
Do the suggestions in [this SO question](https://stackoverflow.com/questions/75574268/missing-file-libarclite-iphoneos-a-xcode-14-3) help? But I'll definitely look into it as well.

**Comment by sderungs99** [comment]
Thanks. Let me try to replicate this again and provide more details about my environment. 

For me, I also created a brand new project, added this dependency only and on 'flutter run' getting that build error I mentioned above.

**Comment by sderungs99** [comment]
Hi, I replicated it again by creating a brand new project. Here is my command line prompt:

```cmd
22:41:18:play % flutter create sim_problem
Signing iOS app for device deployment using developer identity: "Apple
Development: iOS Development (7NCSRB2Z7X)"
Creating project sim_problem...
Resolving dependencies in sim_problem... (1.1s)
Got dependencies in sim_problem.
Wrote 129 files.

All done!
You can find general documentation for Flutter at: https://docs.flutter.dev/
Detailed API documentation is available at: https://api.flutter.dev/
If you prefer video documentation, consider:
https://www.youtube.com/c/flutterdev

In order to run your application, type:

  $ cd sim_problem
  $ flutter run

Your application code is in sim_problem/lib/main.dart.

22:41:38:play % cd sim_problem
22:41:55:sim_problem % flutter run
Launching lib/main.dart on iPhone 14 Pro Max in debug mode...
Running Xcode build...
 └─Compiling, linking and signing...                      2,586ms
Xcode build done.                                           11.0s
[VERBOSE-2:FlutterDarwinContextMetalImpeller.mm(35)] Using the Impeller rendering backend.
Syncing files to device iPhone 14 Pro Max...                        42ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

A Dart VM Service on iPhone 14 Pro Max is available at:
http://127.0.0.1:54960/OMwFHPSqH40=/
The Flutter DevTools debugger and profiler on iPhone 14 Pro Max is available at:
http://127.0.0.1:9100?uri=http://127.0.0.1:54960/OMwFHPSqH40=/

Application finished.


22:42:30:sim_problem % flutter pub add sqlite3_flutter_libs
Resolving dependencies...
  collection 1.17.1 (1.17.2 available)
  matcher 0.12.15 (0.12.16 available)
  material_color_utilities 0.2.0 (0.5.0 available)
  source_span 1.9.1 (1.10.0 available)
+ sqlite3_flutter_libs 0.5.15
  test_api 0.5.1 (0.5.2 available)
Changed 1 dependency!


22:42:52:sim_problem % flutter run
Resolving dependencies...
  collection 1.17.1 (1.17.2 available)
  matcher 0.12.15 (0.12.16 available)
  material_color_utilities 0.2.0 (0.5.0 available)
  source_span 1.9.1 (1.10.0 available)
  test_api 0.5.1 (0.5.2 available)
Got dependencies!
Launching lib/main.dart on iPhone 14 Pro Max in debug mode...
Running pod install...                                             723ms
Upgrading Pods-Runner-frameworks.sh
Running Xcode build...
Xcode build done.                                            2.3s
Failed to build iOS app
Error (Xcode): File not found:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/u
sr/lib/arc/libarclite_iphonesimulator.a


Error (Xcode): Linker command failed with exit code 1 (use -v to see invocation)


Could not build the application for the simulator.
Error launching application on iPhone 14 Pro Max.
22:43:09:sim_problem %
```

Here is my `flutter doctor` output:
```cmd
22:43:09:sim_problem % flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.10.0, on macOS 13.3.1 22E261 darwin-arm64, locale en-SG)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.1)
[✓] VS Code (version 1.78.2)
[✓] Connected device (3 available)
[✓] Network resources

• No issues found!
```

Hopefully this helps to analyse further?

**Comment by sderungs99** [comment]
This issue is resolved with Flutter 3.10.1

As far as I am concerned, issue can be closed.
